packages/adblocker-webextension/adblocker.ts

```diff
@@ -505,6 +551,44 @@ export class WebExtensionBlocker extends FiltersEngine {
           },
     );
   }
+
+  private executeScriptlets(tabId: number, scripts: string[]): void {
+    // the scriptlet code that contains patches for the website
+    const codeRunningInPage = `(function(){
```

Comment:
We might want to use this function as well to wrap the script? https://github.com/ghostery/adblocker/blob/b11cb64f59153becab6f37f2cca4cf1228f11a06/packages/adblocker-content/adblocker.ts#L236

And maybe mimic this for the logic below: https://github.com/ghostery/adblocker/blob/b11cb64f59153becab6f37f2cca4cf1228f11a06/packages/adblocker-content/adblocker.ts#L255